libs/ui/select/brain/src/lib/brn-select.component.ts

```diff
@@ -121,8 +117,24 @@ export class BrnSelectComponent
 	/** Overlay pane containing the options. */
 	@ContentChild(BrnSelectContentComponent)
 	protected selectContent!: BrnSelectContentComponent;
-	@ContentChildren(BrnSelectOptionDirective, { descendants: true })
-	protected options!: QueryList<BrnSelectOptionDirective>;
+
+	protected options = contentChildren(CdkOption, { descendants: true });
+	protected options$ = toObservable(this.options);
+	protected optionsChanged$ = this.options$.pipe(skip(1));
+	/**
+	 * Whenever the CdkOption query is updated we should update the possibleOptions in the select service
+	 */
+	private syncOptions$ = effect(
```

Comment:
can we move this into the constructor just for the sake of consistency?

Reply:
good ideas, yeah I'll do this